backend/app/api/routes/auth.py

```
@@ -0,0 +1,81 @@
+from app.api.dependencies.auth import get_current_user
+from app.core.password import hash_password, verify_password
+from app.core.tokens import create_access_token, create_refresh_token, decode_token
+from app.db.session import get_db
+from app.models.user import User
+from app.schemas.auth import LoginRequest, RefreshRequest, TokenPairResponse, UserMeResponse
+
+router = APIRouter(prefix="/auth", tags=["auth"])
+
+
+@router.post("/login", response_model=TokenPairResponse)
+def login(payload: LoginRequest, db: Session = Depends(get_db)) -> TokenPairResponse:
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)